src/cfnlint/custom_rules.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import cfnlint
+
+
+LOGGER = logging.getLogger(__name__)
+Operators = {'EQUALS': lambda x, y: equals(x, y)}
+
+def check_custom_rules(filename, template):
+    """ Process custom rule file """
+    matches = []
+    with open(filename) as customRules:
+        count = 1
+        for line in customRules:
+            LOGGER.debug('Processing Custom Rule Line %d', count)
+            line = line.replace('"', '')
+            rule = line.split(' ')
+            if len(rule) == 4 and rule[0][0] != '#':
+                try:
+                    result = Operators[rule[2]](rule, template.get_resource_properties([rule[0]]))
```

Comment:
Assigning `rule[2]` and `rule[0]` to variables would really help with readability

Reply:
Changed, minus one reference which happens prior to checking the length - Assigning variables prior to that point could result in errors with invalid inputs